### DISCUSSION TITLE
Can't edit date field when off disabled by conditions

### DISCUSSION THREAD

**Discussion by spitoutzi** [Budibase/budibase#11048] [Help] [answered]
Hi!
I create form fields disabled by default. By pressing the "Edit" button, I write the value "yes" to the variable. The "Disabled" value of form fields is disabled if the variable is "Yes" by Conditions. When I press "Edit" button I can edit text field, but the input in the date/time field is not available. I click on the field, but nothing happens. Without Conditions everything is OK :) Example in the attachment
[Test app-export-1687939382656.tar.gz](https://github.com/Budibase/budibase/files/11891763/Test.app-export-1687939382656.tar.gz)
![image](https://github.com/Budibase/budibase/assets/126158687/3d608fbd-d999-471b-b84f-fddd4a85b8c6)
![image](https://github.com/Budibase/budibase/assets/126158687/1c83a1a3-39a0-429f-8f67-4365ec452d8c)

https://github.com/Budibase/budibase/assets/126158687/806cd394-f439-4066-ac4e-0c3903d3dbd0

**Comment by ConorWebb96** [comment, marked as answer]
Hey @spitoutzi,

Yup this is definitely a bug, I raised it with the engineering team and a fixed has been pr'ed here; https://github.com/Budibase/budibase/pull/11049 

This should work on the next iteration of Budibase. You can check in on the above link to see whenever it gets merged in. 

Thank you for bring this to our attention!
